docs/apis-clients/tasklist-api/mutations/claim-task.mdx

```diff
@@ -5,6 +5,14 @@ title: claimTask
 
 Claim a task with taskId to currently logged in user. Returns the task.
 
+:::note
```

Comment:
Just a quick technical note -- you'll need to adjust "m2m" to "M2M" and "therefor" to "therefore" in these notes 👍

Reply:
I adjusted this in my latest commit, but let me know if you have any questions.